modules/nf-core/trgt/merge/tests/main.nf.test

```diff
@@ -0,0 +1,192 @@
+                input[3] = Channel.of('chr22\t18890357\t18890451\tID=TEST;MOTIFS=AT;STRUC=(AT)n')
+                            .collectFile(name : 'repeats.bed', newLine: false)
+                            .map { file -> [ [ id : 'chr22' ], file ] }
+                            .first()
+                """
+            }
+        }
+        run("BCFTOOLS_SORT"){
+            script "../../../bcftools/sort/main.nf"
+            process {
+                """
+                input[0] = TRGT_GENOTYPE.out.vcf
+                """
+            }
+        }
+    }
+    test("homo sapiens - 2 VCFs") {
+        when {
+            process {
+                """
+                input[0] = BCFTOOLS_SORT.out.vcf
+                            .join(BCFTOOLS_SORT.out.tbi)
+                            .map { meta, vcf, tbi -> [ [ id : 'test' ], vcf, tbi ] }
+                            .groupTuple()
+                input[1] = [[],[]]
+                input[2] = [[],[]]
+                """
+            }
+        }
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    path(process.out.vcf.get(0).get(1)).vcf.variantsMD5,
+                    path(process.out.vcf.get(0).get(1)).vcf.summary,
+                    path(process.out.vcf.get(0).get(1)).vcf.sampleCount,
+                    process.out.versions
+                ).match() }
+            )
+        }
+    }
+    test("homo sapiens - 2 VCFs - reference") {
+        when {
+            process {
+                """
+                input[0] = BCFTOOLS_SORT.out.vcf
+                            .join(BCFTOOLS_SORT.out.tbi)
+                            .map { meta, vcf, tbi -> [ [ id : 'test' ], vcf, tbi ] }
+                            .groupTuple()
+                input[1] = GUNZIP.out.gunzip
+                input[2] = SAMTOOLS_FAIDX.out.fai
+                """
+            }
+        }
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    path(process.out.vcf.get(0).get(1)).vcf.variantsMD5,
+                    path(process.out.vcf.get(0).get(1)).vcf.summary,
+                    path(process.out.vcf.get(0).get(1)).vcf.sampleCount,
+                    process.out.versions
+                ).match() }
+            )
+        }
+    }
+    test("homo sapiens - 1 VCF") {
+        when {
+            process {
+                """
+                input[0] = BCFTOOLS_SORT.out.vcf
+                            .join(BCFTOOLS_SORT.out.tbi)
+                            .first()
+                input[1] = GUNZIP.out.gunzip
+                input[2] = SAMTOOLS_FAIDX.out.fai
+                """
+            }
+        }
+        then {
+            assertAll(
+                { assert !process.success }
+            )
+        }
+    }
+    test("homo sapiens - 1 VCF - --force-single") {
+        config "./force_single.config"
```

Comment:
```suggestion
        config "./nextflow.config"
```

rather than a separate config file. plz see [this way](https://nf-co.re/blog/2024/maintainers-minutes-2024-10-25#new-way-of-passing-parameters-to-modules-within-nf-test-scripts) of passing params to a test from the nextflow config